BUILD.bazel

```
@@ -16,6 +17,18 @@ exports_files([
     "mainnet-canisters.json",
 ])
 
+jq_dep(
+    name = "mainnet_nns_revision",
```

Let's make this name more explicit so as to not confuse this with the revision of the NNS canisters:
```suggestion
    name = "mainnet_nns_subnet_revision",
```